pkg/executables/cmk.go

```diff
@@ -26,17 +27,71 @@ type Cmk struct {
 	writer     filewriter.FileWriter
 	executable Executable
 	config     CmkExecConfig
+	zones []cmkZone
+	domain cmkDomain
+	account cmkAccount
+
 }
 
-// TODO: Add support for domain, account filtering
-func (c *Cmk) ValidateTemplatePresent(ctx context.Context, domain string, zone v1alpha1.CloudStackResourceRef, account string, template v1alpha1.CloudStackResourceRef) error {
-	var filterArg string
+func (c *Cmk) ValidateDatacenterConfig(ctx context.Context, datacenterConfig v1alpha1.CloudStackDatacenterConfig) error {
```

Comment:
these functions introduce some business logic which should exist outside cmk, and instead live in cloudstack.go or validator.go. Are you just putting it here as a placeholder?

Reply:
yes, as a place holder since our cloudstack.go and validator have not been merged yet. This logic will be moved to them later.

Reply:
For something like this you might want to declare your intention explicitly by creating a `to_be_moved.go` and just using functions rather than polluting the `Cmk` struct.

Reply:
I removed these two functions for this PR since they are not used for now. It will add back in coming commit when CloudMonkey wrapper is added.